include/mbedtls/bignum.h

```diff
@@ -45,6 +45,7 @@
 #define MBEDTLS_ERR_MPI_DIVISION_BY_ZERO                  -0x000C  /**< The input argument for division is zero, which is not allowed. */
 #define MBEDTLS_ERR_MPI_NOT_ACCEPTABLE                    -0x000E  /**< The input arguments are not acceptable. */
 #define MBEDTLS_ERR_MPI_ALLOC_FAILED                      -0x0010  /**< Memory allocation failed. */
+#define MBEDTLS_ERR_MPI_TRNG_POSSIBLY_FAULTY              -0x0012  /**< Memory allocation failed. */
```

Comment:
When you add an error code, you need to run `scripts/generate_errors.pl`. Here it complains that 0x0012 is duplicated. You need to pick a different value and add it to `include/mbedtls/error.h` manually. Note that there are strong constraints on error codes, documented tersely in this file.

Reply:
The values are updated manually and I wouldn't be surprised if they're sometimes not up-to-date. Please update the Nr column when you add an error. If some values are wrong, please fix them in a separate commit while you're at it.

Given that there are no more available values on either side of the current MPI range, we need a second range, like there is for GCM and others.

Reply:
Thanks , I corrected the Nr value, and set a second range for the new value.